src/datanode/src/datanode.rs

```diff
@@ -47,6 +47,35 @@ impl Default for ObjectStoreConfig {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct WalConfig {
+    // wal directory
+    pub dir: String,
+    // wal file size in bytes
+    pub file_size: usize,
```

Comment:
Could we use crates like [bytesize](https://github.com/hyunsik/bytesize) to serialize/deserialize the size?
```suggestion
    pub file_size: ByteSize,
```

Reply:
Looks like the `serde` feature is still under development: https://github.com/hyunsik/bytesize/pull/29

Reply:
I copy `ReadableSize` from [raft-engine](https://github.com/tikv/raft-engine/blob/8dd2a39f359ff16f5295f35343f626e0c10132fa/src/util.rs#L23) to make size config items human-readable.